java/__init__.py

```diff
@@ -90,6 +90,8 @@ def java_types_decode(key):
     "List_DiscoveryStrategyConfig": "java.util.List<com.hazelcast.client.impl.protocol.codec.holder.DiscoveryStrategyConfigHolder>",
     "DiscoveryConfig": "com.hazelcast.client.impl.protocol.codec.holder.DiscoveryConfigHolder",
     "WanSyncConfig": "com.hazelcast.client.impl.protocol.codec.holder.WanSyncConfigHolder",
+    "ResourceDefinition": "com.hazelcast.client.impl.protocol.task.dynamicconfig.ResourceDefinitionHolder",
+    "List_ResourceDefinition": "java.util.List<com.hazelcast.client.impl.protocol.task.dynamicconfig.ResourceDefinitionHolder>"
```

Comment:
can you add similar stuff for other languages ? You need to add something like ResourceDefinition: "NA" for them to be able to not break them.

Reply:
Based on definitions for `WanSyncConfig`, this commit should cover the missing pieces? Let me know if not: https://github.com/hazelcast/hazelcast-client-protocol/pull/490/commits/f9b5c4bad426d6b56382b9d150fb8bdf1c05f91e

Reply:
Sure, I am not sure which ones needs to be added anyway but at least we need to test all languages generation. (and I did it, codecs that needs the missing types are ignored) so it is a approve from me